### PR TITLE
Add a test for #1366

### PR DESCRIPTION
**Pull request by mdiep**

### CODE REVIEW

File: Source/CarthageKitTests/ProjectSpec.swift [hunk 1]
```diff
@@ -16,43 +16,69 @@ import Tentacle
 class ProjectSpec: QuickSpec {
 	override func spec() {
 		let directoryURL = NSBundle(forClass: self.dynamicType).URLForResource("CartfilePrivateOnly", withExtension: nil)!
```
Comment (ikesyo) [inline Source/CarthageKitTests/ProjectSpec.swift]: Looks like this should be moved into `it("should load a combined Cartfile when only a Cartfile.private is present")` now.

**Comment by ikesyo** [comment]
Looks good, just a minor note. 👍

**Comment by mdiep** [comment]
Done! ✨

**Comment by ikesyo** [comment]
🎉 🚢

**Pull request merged**